crypto/encrypt.go

```diff
@@ -36,6 +36,7 @@ func GenerateTwofishKey() (key TwofishKey, err error) {
 // NewCipher creates a new Twofish cipher from the key.
 func (key TwofishKey) NewCipher() cipher.Block {
 	// NOTE: NewCipher only returns an error if len(key) != 16, 24, or 32.
+	//       Assumption tested in TestTwofishNewCipherAssumption.
```

Comment:
oh I think I misuderstood what you meant by 'tested the assumption'.

I'm guessing it's okay to not check the error here, since encryption is just math. Comment is still not needed.